platform/android/app/src/androidTest/kotlin/coop/polypoly/polypod/FeatureTest.kt

```diff
@@ -0,0 +1,82 @@
+package coop.polypoly.polypod
+
+import androidx.compose.ui.test.junit4.AndroidComposeTestRule
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performScrollTo
+import androidx.preference.PreferenceManager
+import androidx.test.espresso.web.sugar.Web.onWebView
+import androidx.test.espresso.web.webdriver.DriverAtoms
+import androidx.test.espresso.web.webdriver.Locator
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import coop.polypoly.core.FeatureCategoryId
+import coop.polypoly.polypod.core.UpdateNotification
+import coop.polypoly.polypod.features.FeatureStorage
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+private typealias MainTestRule =
+    AndroidComposeTestRule<ActivityScenarioRule<MainActivity>, MainActivity>
+
+private fun onFeature() =
+    onWebView().inWindow(DriverAtoms.selectFrameByIndex(0))
```

Comment:
Normally I'd import `DriverAtoms.selectFrameByIndex` and the others directly, but it appears IntelliJ currently insists on adding a asterisk import when importing three or more things from one package - seemed like the wrong time to mess with the settings, but I could.

Reply:
Go to `coding style / kotlin / imports` and select `use single name import`.
<img width="1435" alt="Screenshot 2022-09-08 at 15 38 44" src="https://user-images.githubusercontent.com/25011107/189123782-2129a8ea-3e78-42b7-9da4-949376e5c0aa.png">

Reply:
If those changes aren't being committed, I'd rather change the ktlint rules - otherwise everybody working with the project has to fiddle with the settings first. Guess my workaround will do for now.